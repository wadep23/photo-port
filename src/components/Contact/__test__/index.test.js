import { render, cleanup, getByTestId } from '@testing-library/react';
import Contact from '../index';
import '@testing-library/jest-dom/extend-expect';



afterEach(cleanup);

describe('contact form renders', () => {
    it('renders', () => {
        render(<Contact />);
    })

    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Contact />);
        
        expect(asFragment()).toMatchSnapshot();
    })

    // it('matches "Contact Me" on h1 tag', () => {
    //     const wrapper = mount(<h1 data-testid='Contact Me' />);

    //     expect(data-testid.value)
    // })
})