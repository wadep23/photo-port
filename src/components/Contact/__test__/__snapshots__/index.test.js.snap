// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact form renders matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="Contact Me"
    >
      Contact Me
    </h1>
    <form
      id="contact-form"
    >
      <div>
        <label
          for="name"
        >
          Name: 
        </label>
        <input
          name="name"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          for="email"
        >
          Email: 
        </label>
        <input
          name="email"
          type="email"
          value=""
        />
      </div>
      <div>
        <label
          for="message"
        >
          Message: 
        </label>
        <textarea
          name="message"
          rows="5"
        />
      </div>
      <button
        type="submit"
      >
        Submit
      </button>
    </form>
  </section>
</DocumentFragment>
`;
